src/infrastructure/organisations/api.js

```diff
@@ -12,17 +12,15 @@ const callApi = async (method, path, correlationId, body) => {
   const basePathSeperator = hasSeperator ? '' : '/';
   const opts = {
     method,
-    uri: `${config.organisations.service.url}${basePathSeperator}${path}`,
     headers: {
       authorization: `bearer ${token}`,
       'x-correlation-id': correlationId,
     },
-    json: true,
   };
   if (body && (method === 'POST' || method !== 'PUT' || method !== 'PATCH')) {
     opts.body = body;
   }
-  return rp(opts);
+  return fetchApi(`${config.organisations.service.url}${basePathSeperator}${path}`, opts);
```

Comment:
could uri be not part options anymore?

Reply:
URI is no longer a part of options because it isn't implemented with `request-promise` anymore but rather the `fetch` function. This resembles how this would look if we remove our wrapper and use native `fetch` directly.